site/src/components/BuildsTable/BuildRow.tsx

```diff
@@ -0,0 +1,139 @@
+import { makeStyles } from "@material-ui/core/styles"
+import TableCell from "@material-ui/core/TableCell"
+import TableRow from "@material-ui/core/TableRow"
+import { WorkspaceBuild } from "api/typesGenerated"
+import { Stack } from "components/Stack/Stack"
+import { useTranslation } from "react-i18next"
+import { MONOSPACE_FONT_FAMILY } from "theme/constants"
+import {
+  displayWorkspaceBuildDuration,
+  getDisplayWorkspaceBuildInitiatedBy,
+} from "util/workspace"
+import { BuildAvatar } from "./BuildAvatar"
+
+export interface BuildRowProps {
+  build: WorkspaceBuild
+}
+
+export const BuildRow: React.FC<BuildRowProps> = ({ build }) => {
+  const styles = useStyles()
+  const { t } = useTranslation("workspacePage")
+  const initiatedBy = getDisplayWorkspaceBuildInitiatedBy(build)
+
+  return (
+    <TableRow
+      hover
+      data-testid={`build-${build.id}`}
+      className={styles.buildRow}
+    >
+      <TableCell className={styles.buildCell}>
+        <Stack
+          direction="row"
+          alignItems="center"
+          className={styles.buildRow}
+          tabIndex={0}
+        >
+          <Stack
+            direction="row"
+            alignItems="center"
+            justifyContent="space-between"
+            className={styles.buildRowMain}
+          >
+            <Stack direction="row" alignItems="center">
+              <BuildAvatar build={build} />
+              <div>
+                <Stack
+                  className={styles.buildResume}
+                  direction="row"
+                  alignItems="center"
+                  spacing={1}
+                >
+                  <span>
+                    <strong>{initiatedBy}</strong>{" "}
```

Comment:
We can add translations with formatting by using a [`Trans` component](https://react.i18next.com/latest/trans-component). 
Presley asked me to write out an example, so there's [one in our how-tos](https://www.notion.so/coderhq/V2-Frontend-Code-Conventions-and-Best-Practices-024466624dfa482e894570e95a6cc842#b22e2d30e65e4c899022fb0b6b1a7d32).

Reply:
I found it hard to use `Trans` here because of the conditionals but I moved the text to translation files.